browser-tests/nightwatch_runner.js

```diff
@@ -75,7 +75,7 @@ module.exports = {
             .frame('articleContent')
             // Check the text in the article "Ray Charles"
             .useXpath()
-            .waitForElementPresent("//div[@id='content']/div[@id='mw-content-text']/h2[@id='mweQ']", 40000)
+            .waitForElementPresent("//div[@id='content']/div[@id='mw-content-text']/h2[@id='mweQ']", 20000)
```

Comment:
Revert these changes, which were done in an attempt to fix the tests, and see if reverting them destabilizes currently working tests.

Reply:
Reverting causes test failure.